lib/MetaCPAN/Web/Model/API/Favorite.pm

```diff
@@ -135,7 +135,10 @@ sub find_plussers {
     my $total_authors = @plusser_details;
 
     # find total non pauseid users who have ++ed the dist.
-    my $total_nonauthors = ( $total_plussers - $total_authors );
+    my $total_nonauthors
+        = ( $total_plussers - $total_authors )
+        ? ( $total_plussers - $total_authors ) > 0
```

Comment:
save a boolean (whether or not this calculation is greater than zero) to the variable.

I think that's not what you meant.